### PR TITLE
Support for database migrations using Alembic

### PR DESCRIPTION
**Pull request by simark**

### CODE REVIEW

File: src/canadiantracker/triangle.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def pretty_name(self) -> str:
 
 
 class ProductInventory(Iterable):
-    def __init__(self):
+    def __init__(self, dev_max_categories=0, dev_max_pages_per_category=0):
```
Comment (jgalar) [inline src/canadiantracker/triangle.py]: Please use type annotations.

There are a couple missing in the current code, but I'm trying to enforce this.

**Review by jgalar — CHANGES_REQUESTED** [review]
Otherwise the PR looks good. Thanks a lot for the detailed messages.

File: src/canadiantracker/triangle.py (hunk 1 above)
Comment (simark) [reply src/canadiantracker/triangle.py]: Ok, fixed (I think).

Comment (jgalar) [reply src/canadiantracker/triangle.py]: Fantastic, thanks!

**Pull request merged**